apps/web/src/components/organization-select.tsx

```diff
@@ -42,31 +42,45 @@ import {
   FormMessage,
 } from "./ui/form";
 import { Input } from "./ui/input";
+import { useToast } from "./ui/use-toast";
+
+
+
 
-type OrganizationForm = Omit<InsertOrganization, "owner">;
 
 export default function OrganizationSelect() {
   const pathname = usePathname();
-
   const organizationFromPathname = decodeURIComponent(
     // @ts-expect-error Since we are taking something from a pathname there has to be something
     pathname.split("/").at(2),
   );
```

Comment:
We can take the orgname from `props.params.name`

Reply:
not here, there would be no name since this component is used in navbar

Reply:
Em, no it won't.